decksite/achievements.py

```diff
@@ -103,6 +102,14 @@ def load_summary(self, season_id: Optional[int] = None) -> Optional[str]:
             players_text = ngettext('1 player', f'%(num)d players', res.pnum)
             return f'Earned{times_text} by {players_text}.'
         return None
+    def percent(self, season_id: Optional[int] = None) -> float:
+        season_condition = f'season_id = {season_id}' if season_id != 'all' else ''
```

Comment:
This was the 500 on /seasons/all/achievements – we have `query.season_query(season_id)` to encapsulate this logic.

Reply:
This was me blindly copying the code above after merging with head at midnight

Reply:
c8e43f7d fixes the above call.